bootcamp/services/user_service.py

```diff
@@ -0,0 +1,42 @@
+import uuid
+
+from tornado.gen import coroutine, Return
+
+from bootcamp.services import logger
+from bootcamp.services.datastores.user_store import UserStore
+
+
+class UserService(object):
+    def __init__(self):
+        self.store = UserStore()
+
+    @coroutine
+    def create_user_with_entity(self, user_entity):
+        if not user_entity.uuid:
+            user_entity.uuid = str(uuid.uuid4())
+        user_entity.validate()
+
+        user = yield self.store.create_from_entity(user_entity)
```

Comment:
User name should be unique, and we should do user_name dedup before creating.
Let's do it in a separate diff.